content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -19,136 +16,129 @@ deploy Kubeflow on Amazon Web Services (AWS).
+    ```shell
+    # Download config files
+    export CONFIG="/tmp/kfctl_aws.yaml"
+    wget https://raw.githubusercontent.com/kubeflow/kubeflow/master/bootstrap/config/kfctl_aws.yaml -O ${CONFIG}
+    ```
 
+    * `kfctl_aws.yaml` is one of setup manifests, please check [kfctl_aws_cognito.yaml](https://github.com/kubeflow/kubeflow/blob/master/bootstrap/config/kfctl_aws_cognito.yaml) for the template to enable authentication.
 
-## Kubeflow installation
 
-1. Run the following commands to download the latest `kfctl.sh`
+    - If you plan to use `kfctl` to create a new eks cluster, please remove follow lines in the manifest file.
+    - If you want to install on existing EKS cluster, please change roles to your worker node group roles.
 
     ```shell
-    export KUBEFLOW_SRC=/tmp/kubeflow-aws
-    export KUBEFLOW_TAG=v0.5-branch
-
-    mkdir -p ${KUBEFLOW_SRC} && cd ${KUBEFLOW_SRC}
-    curl https://raw.githubusercontent.com/kubeflow/kubeflow/${KUBEFLOW_TAG}/scripts/download.sh | bash
+    roles:
+      - eksctl-kubeflow-aws-nodegroup-ng-a2-NodeInstanceRole-xxxxxxx
     ```
 
-    * KUBEFLOW_SRC - Full path to your preferred download directory. Please use the full absolute path, for example `/tmp/kubeflow-aws`
-
 1. Run the following commands to set up your environment and initialize the cluster.
+    Since there're many ways to create your cluster, we highly recommend you to get our own eks cluster ready.
 
     > Note: If you would like to install Kubeflow on your existing EKS cluster, please skip this step
     > and follow the setup instructions for an [existing cluster](/docs/aws/deploy/existing-cluster) instead.
     > When you are finished, return here and resume with the next step.
 
 
     ```shell
-    export KFAPP=kfapp
-    export REGION=us-west-2
-    export AWS_CLUSTER_NAME=kubeflow-aws
+    export KFAPP=kfaws
+
+    kfctl init ${KFAPP} --config=${CONFIG} -V
+    cd ${KFAPP}
 
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform aws \
-    --awsClusterName ${AWS_CLUSTER_NAME} \
-    --awsRegion ${REGION}
+    kfctl generate all -V
+    kfctl apply all -V
     ```
 
+    * KFAPP - Use a relative directory name here rather than absolute path, such as `kfapp`. It will be used as eks cluster name.
+    * CONFIG - Path to the configuration file
 
-    * AWS_CLUSTER_NAME - Specify a unique name for your Amazon EKS cluster.
-    * KFAPP - Use a relative directory name here rather than absolute path, such as `kfapp`.
-    * REGION - Use the AWS Region you want to create your cluster in.
 
-1. Generate and apply platform changes.
+    __*Important!!!*__ By default, these scripts create an AWS Application Load Balancer for Kubeflow that is open to public. This is good for development testing and for short term use, but we do not recommend that you use this configuration for production workloads.
```

Comment:
I think the bold text is enough and you can remove all the explanation marks. :)

Reply:
Make it